apps/docs/docs/features/inquirer.md

```diff
@@ -40,8 +40,7 @@ export class TaskRunner implements CommandRunner {
   ): Promise<void> {
     let task = inputs[0];
     if (!task) {
-      task = (await this.inquirer.ask<{ task: string }>('task', undefined:w
-      ))
```

Comment:
Ha! Good catch! That's what I get for using a vim plugin. This is all that's needed to change. It'll deploy up to the docs through docusaurus